### ISSUE TITLE
fail to build on ubuntu 15.04

### ISSUE THREAD

**Issue by wodeqiangne** [allenai/pdffigures#10]
with g++4.9, i fail to build because of failed to find PDFDoc.h
i've apt-get install libpoppler-dev libleptonica-dev

**Comment by chrisc36** [comment]
I had not tested pdffigures on Ubuntu 15.04 before, but I was able to get everything working with a few tweaks.

The initial problem was fixed with
"apt-get install libpoppler-private-dev", it looks like some of the required header files are in the libpoppler-private-dev package on ubuntu 15.04's package system.

I also found that apt-get was loading poppler version 0.30 on ubuntu 15.04 which had a few incompatibilities with the current code base, I was able to iron them out with only a few changes so current branch should build against both poppler 0.30 and 0.37. 

So in short pulling and running "apt-get install libpoppler-private-dev" should allow you to compile.

**Comment by wodeqiangne** [comment]
thanks, this means a lot

**Comment by jenniferzhu** [comment]
This is the key to solve my issues! I think we should put this line on the readme installation session. @chrisc36
